tests/unit/test_github_session.py

```diff
@@ -146,15 +151,43 @@ def test_token_auth_disables_basic_auth(self):
         s = self.build_session()
         s.auth = ('foo', 'bar')
         s.token_auth('token goes here')
-        assert s.auth is None
+        assert s.auth != ('foo', 'bar')
 
     def test_token_auth_does_not_use_falsey_values(self):
         """Test that token auth will not authenticate with falsey values"""
         bad_tokens = [None, '']
+        req = requests.Request('GET', 'https://api.github.com/')
         for token in bad_tokens:
             s = self.build_session()
             s.token_auth(token)
-            assert 'Authorization' not in s.headers
+            pr = s.prepare_request(req)
+            assert 'Authorization' not in pr.headers
+
+    def test_token_auth_with_netrc_works(self, tmpdir):
+
```

Comment:
Most tests in this file have a docstring, please add one here. It doesn't need to be one-line.

Reply:
Sorry, will update